models/stg_iterable__event_extension.sql

```diff
@@ -24,46 +24,68 @@ fields as (
                 staging_columns=get_event_extension_columns()
             )
         }}
+
+        --The below script allows for pass through columns.
+        {% if var('iterable_event_extension_pass_through_columns') %} 
+        ,
+        {{ var('iterable_event_extension_pass_through_columns') | join (", ")}}
+
+        {% endif %}
```

Comment:
Thanks for this-- I had been testing both the old and new but ran into trouble with the new, but that was because I hadn't added the necessary macro in the get_columns files. 

I updated with the new version and tested out the alias feature too.

In the transforms, I modified `iterable__list_user_history` to add the macro, and for the below models:
1. For the `iterable__user`, there's a group-by so I think best to keep it as is
2. `iterable__events` already does a select * from stg_iterable__event_extension